### PR TITLE
Check new search 4xx

### PR DESCRIPTION
**Pull request by TPRobots**
Pull request opened by github-pullrequestcreator.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,179 @@
-__pycache__
-.python-version
+
```
Comment (sloev) [inline .gitignore]: lets add `.env` here just for good measure

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+description = "Parses query args in sanic using type annotations"
+readme = "README.md"
+keywords = ["sanicargs", "sanic", "query", "args", "type annotations"]
+authors = ["Johannes Valbjørn <jgv@trustpilot.com>",  "Thomas Thiebaud <tht@trustpilot.com>"]
+repository = "https://github.com/trustpilot/python-sanicargs"
+classifiers= [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3.6",
```
Comment (sloev) [inline pyproject.toml]: add 3.7 and 3.8 here as well

File: .travis.yml [hunk 3]
```diff
@@ -1,52 +1,26 @@
...
+  - poetry run pytest -vvv tests
+  - poetry build
 
 deploy:
-  provider: releases
-  api_key:
-    secure: E9Q27QCUrW2qspYAhwy7kIwt+TENdr6yhBEIlTGd6s2QaF9KmBn7d7ouLQ+93sV/yISajArlagblPifZLm9mXnvs5BeIA7Dy/EbXgYN1b62iRpOqGCKZ4PaUino8fQXdKgcqWEovIC7PpER/h0g0o2nPULqHADohL2CsGeVXGekfRuvffnQrbcJcfsJtS3Px+u39OhU2s/o718U8XNShDT4OZgJBkn0V+1Toc3ZwBwsZs9bvLVC4keJHTKNwGwG1CyZOzdrp/TizZdrXtjpnA9Dk8jo2Fa1DPDGAnG8ORogz722RCou5PHVQiRnrbRPOuyhPYZe5R97NBi1omMuKXRneLyxUSqfz09nON4Jbah7qb3wtgUX7dSV1XdiBx/mB9bDiIc85RqW+FQJZqxvZU76JDrccPIricaHM05Md/HXtRYAFmcgmhRrwifW2++NRiosX66HocmdyIMQj9TNhIOAe8IeBVub011b++jlEzbN1xHEO26zGGHN9SHfNlj8nyWhojvU56pEv2w/0DSJyRiIbAdTet03u2uF1nEPYJEZB81sJjhrq1vY5aMibCRmElUzmo1RIrrE6a3vy5bFL6unYUFPmcO4hfWDj0TYpnfUkB73Vjjgv83eUM1MW2DoA4AgUk7dl1BE5kRXdO0a9FAqo9MBXNKRAeScdmC/rIkE=
-  file: dist/*
+  provider: script
+  script: poetry publish
```
Comment (sloev) [inline .travis.yml]: cant do that without pypi credentials...

Comment (sloev) [reply .travis.yml]: solved by this comment:
https://trustpilot.slack.com/archives/GAGH48Z2R/p1584533530014000?thread_ts=1584529878.012200&cid=GAGH48Z2R

Comment (thomasthiebaud) [reply .travis.yml]: It will come from travis https://python-poetry.org/docs/repositories/#configuring-credentials

File: sanicargs/__init__.py [hunk 4]
```diff
@@ -55,26 +50,26 @@ def parse_query_args(func):
...
-        (name, p.annotation, p.default)
-        for name, p in notations.parameters.items()
+        (name, p.annotation, p.default) for name, p in notations.parameters.items()
     ]
     request_arg_name = inspect.getfullargspec(func)[0][0]
-    
+
     @wraps(func)
     async def inner(request, *old_args, **route_parameters):
+        print(request.args)
```
Comment (sallas) [inline sanicargs/__init__.py]: you forgot a print? :)

Comment (thomasthiebaud) [reply sanicargs/__init__.py]: Ha, yes. I'll fix that

Comment (sloev) [reply sanicargs/__init__.py]: very much got into pypi thx to me 🤦‍♂ 
i will: 

1. delete release on pypi
2. remove the print statement
3. bump minor version and write history
4. make a new release

aka: fix the mess i created :-)

**Pull request merged**